lambda-samples/samples/docdb-scaleOnSchedule.py

```diff
@@ -0,0 +1,92 @@
+import json
+import os
+import boto3
+import random
+import logging
+import string
+import logger
+import sys
+
+#Set up the logger
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+clusterIdentifier = os.environ['CLUSTER_IDENTIFIER']
+instanceSuffix = os.environ['INSTANCE_NAME_SUFFIX']
```

Comment:
Add validation, as instance names must begin with a letter; must contain only ASCII letters, digits, and hyphens; and must not end with a hyphen or contain two consecutive hyphens.

Reply:
Updated script to used a fixed suffix, instance name suffix is not taken from user input